src/cython/vapoursynth.pyx

```diff
@@ -1225,6 +1267,47 @@ cdef FrameProps createFrameProps(RawFrame f):
 Mapping.register(FrameProps)
 
 
+cdef class ChannelLayout(int):
+    cdef readonly int num_channels
```

Comment:
Not required.

Reply:
All right, so we only use the channel layout and can put this on public API